src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -200,6 +200,7 @@ export class UiSettingsClient implements IUiSettingsClient {
     changes: Record<string, any>;
     autoCreateOrUpgradeIfMissing?: boolean;
   }) {
+    changes = this.translateChanges(changes, 'timeline', 'timelion');
```

Comment:
Can we define these as constants since they're used in multiple places?

Reply:
This one I considered, but really shouldn't be used besides the two places that they are used.